### PR TITLE
UIA ButtonWrapper 

### PR DESCRIPTION
**Pull request by airelil**
UIA ButtonWrapper support for radio buttons 'select', 'is_selected'
UIA Wrapper return self where possible, this is to allow actions chaining.
WpfApplication1 with more controls
Handle query interface exceptions in uia_defines.get_elem_interface

### CODE REVIEW

**Comment by vasily-v-ryabov** [comment]
Looks fine to me. :+1: 
Just in doubt about `iface = None`. It will lead to confusing exception like `NoneType object has no ### attribute`. Probably explicit `PatternNotSupported` custom exception would be more clear.

**Comment by vasily-v-ryabov** [comment]
I see almost every UIAWrapper test case takes about 1 minute! Previous build shows much better times.

**Comment by airelil** [comment]
@vasily-v-ryabov, I also noticed that UIA_SUPPORT=YES runs 40 mins vs. UIA_SUPPORT=NO. I tried to compare the performance of the same UIAWrapperTests.testParent before all my changes and after. These are the profiler outputs:
Commit 406468b
```
         66266 function calls (64506 primitive calls) in 3.118 seconds

   Ordered by: internal time

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
        1    2.000    2.000    2.000    2.000 {win32event.WaitForSingleObject}
        6    0.545    0.091    0.545    0.091 UIAElementInfo.py:171(children)
        1    0.090    0.090    0.090    0.090 {time.sleep}
        1    0.067    0.067    0.067    0.067 {win32event.WaitForInputIdle}
      177    0.034    0.000    0.034    0.000 __init__.py:1170(Release)
        1    0.034    0.034    0.043    0.043 automation.py:2(<module>)
        3    0.015    0.005    0.016    0.005 UIAElementInfo.py:183(descendants)
```

Commit 5be9bdf
```
         146146 function calls (144300 primitive calls) in 3.532 seconds

   Ordered by: internal time

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
        1    2.001    2.001    2.001    2.001 {win32event.WaitForSingleObject}
        6    0.466    0.078    0.467    0.078 UIAElementInfo.py:170(children)
     2114    0.131    0.000    0.138    0.000 UIAElementInfo.py:146(class_name)
     1924    0.116    0.000    0.116    0.000 UIAElementInfo.py:156(handle)
      343    0.092    0.000    0.092    0.000 __init__.py:1170(Release)
        1    0.090    0.090    0.090    0.090 {time.sleep}
        1    0.068    0.068    0.068    0.068 {win32event.WaitForInputIdle}
        3    0.053    0.018    0.053    0.018 UIAElementInfo.py:182(descendants)
```

File: pywinauto/unittests/test_UIAWrapper.py [hunk 1]
```diff
@@ -131,6 +131,16 @@ def testTypeKeys(self):
...
+            elem = self.button.element_info.element
+            self.assertRaises(
+                    uia_defs.NoPatternInterfaceError,
+                    uia_defs.get_elem_interface,
+                    elem, 
+                    "Selection",
+                    )
+
         def testGetProperties(self):
             uia_props = {'class_name',
```
Comment (vasily-v-ryabov) [inline pywinauto/unittests/test_UIAWrapper.py]: It doesn't work in Py2.6. Should be `set(['class_name', ... ])`.

**Pull request merged**